packaging/nuget/package.ps1

```diff
@@ -0,0 +1,26 @@
+Push-Location $PSScriptRoot
+[Environment]::CurrentDirectory = $PWD
+
+$ProjectsDir = "..\..\src"
```

Comment:
In other scripts we tend to import https://github.com/dotnet/cli/blob/master/scripts/_common.ps1 and make paths relative to the $RepoRoot variable